### PR TITLE
Improve/quotes

### PR DESCRIPTION
**Pull request by fallaciousreasoning**
Adds improved quote handling, so we don't output things like
- dont
- wont
- wouldnt

ect

### CODE REVIEW

File: markov.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const fixCapitalisation = sentences => {
                 case 'i':
                     sentences[i][j] = 'I';
                     break;
-                case 'ill':
+                case 'i\'ll':
```
Comment (drkno) [inline markov.js:9]: Would double quotes be better?

File: markov.js [hunk 2]
```diff
@@ -16,11 +16,13 @@ const fixCapitalisation = sentences => {
...
 const buildChains = (corpus, chainLength) => {
-    // Break the corpus into sentences and only take sentences with more than 10
-    // characters.
-    corpus = corpus.toLowerCase().replace("\n", ".");
-
-    let sentences = corpus.split(/[\.!\?\n]/).filter(sentence => sentence.length > 10);
+    // Convert the corpus to lower case and split on terminator characters,
+    // ensuring that each sentence has at least 10 characters so we don't end
+    // up with weird stumpy sentences
+    let sentences = corpus
```
Comment (drkno) [inline markov.js]: sentences isn't reassigned so can be const

File: markov.js [hunk 3]
```diff
@@ -33,19 +35,20 @@ const buildChains = (corpus, chainLength) => {
...
 const parseSentence = (sentence, chainLength, chains) => {
     // We only want normal people characters (ASCII because we're racist).
-    sentence = sentence.replace(/[^a-z^ ]/g, "");
+    // a-z and ' chars that are surrounded by letters
+    sentence = sentence
+        .replace(/[^a-z^ ^']|/g, '')
+        .replace(/ '|' /g, '');
 
-    let tokens = sentence.split(" ").filter(token => token.length > 0);
+    let tokens = sentence.split(' ').filter(token => token.length > 0);
```
Comment (drkno) [inline markov.js]: const

File: markov.js [hunk 4]
```diff
@@ -108,6 +111,10 @@ class MarkovGenerator {
             sentence.push(word);
         }
 
+        if (sentence.length > 0) {
+            sentences.push(sentence);
+        }
```
Comment (drkno) [inline markov.js:116]: my bad

File: markov.js (hunk 1 above)
Comment (fallaciousreasoning) [reply markov.js:9]: But. But then they'd be different. I want consistency!

File: markov.js (hunk 4 above)
Comment (fallaciousreasoning) [reply markov.js:116]: All g

File: markov.js (hunk 3 above)
Comment (fallaciousreasoning) [reply markov.js]: Fixed

File: markov.js (hunk 2 above)
Comment (fallaciousreasoning) [reply markov.js]: Fixed

**Pull request merged**